src/script/page/AppLock.test.ts

```diff
@@ -49,18 +51,56 @@ class AppLockPage extends TestPage<AppLockProps> {
   getForm = () => this.get('form');
   formSubmit = () => this.submit(this.getForm());
 }
+
+const clientRepository = ({} as unknown) as ClientRepository;
+
+const createTeamState = ({
+  status = 'enabled',
+  enforceAppLock = false,
+  inactivityTimeoutSecs = 10,
+}: {
+  enforceAppLock?: boolean;
+  inactivityTimeoutSecs?: number;
+  status?: string;
+} = {}) => {
+  const teamState = new TeamState();
+  jest.spyOn(teamState, 'isTeam').mockImplementation(ko.pureComputed(() => true));
+  const teamFeatures = {
+    applock: {
+      config: {
+        enforceAppLock,
+        inactivityTimeoutSecs,
+      },
+      status,
+    },
+  };
+  jest.spyOn(teamState, 'teamFeatures').mockImplementation(ko.observable(teamFeatures));
+  return teamState;
+};
+
+const createAppLockState = (teamState?: TeamState) => {
+  teamState = teamState ?? createTeamState();
+  const appLockState = new AppLockState(teamState);
+  return appLockState;
+};
+
+const createAppLockRepository = (appLockState?: AppLockState) => {
+  const userState = new UserState();
+  appLockState = appLockState ?? createAppLockState();
+  jest.spyOn(userState, 'self').mockImplementation(ko.observable({id: createRandomUuid()}));
+  const appLockRepository = new AppLockRepository(userState, appLockState);
+  return appLockRepository;
+};
 describe('AppLock', () => {
   describe('disabled feature', () => {
     it('does not shows up if applock is disabled', () => {
-      const teamState: Partial<TeamState> = {
-        appLockInactivityTimeoutSecs: ko.pureComputed(() => 10),
-        isAppLockEnabled: ko.pureComputed(() => false),
-        isAppLockEnforced: ko.pureComputed(() => false),
-      };
+      const appLockState = createAppLockState(createTeamState({status: 'disabled'}));
```

Comment:
```suggestion
      const appLockState = createAppLockState(createTeamState({status: FeatureStatus.DISABLED}));
```

from '@wireapp/api-client/src/team'